app/views/content/train-to-be-a-teacher/what-is-a-pgce.md

```diff
@@ -10,6 +11,9 @@ related_content:
 date: "2021-11-01"
 image: "static/content/hero-images/0015.jpg"
 backlink: "../../"
+navigation: 20.55
+navigation_title: What is a PGCE
+navigation_description: Find out what a postgraduate certificate in education (PGCE) is and if you need one.
```

Comment:
As per the QTS suggestion, I think we need to try and get across that PGCE is additional rather than being what you need as a standalone qualification

My suggested description isn't quite right yet though!

```suggestion
navigation_description: Having a PGCE in addition to QTS can help you develop your teaching skills and knowledge.
```